### ISSUE TITLE
more seeds in one txt

### ISSUE THREAD

**Issue by CryptoRecoveryWorldwideCZ** [LizardX2/BTCMiner#4]
Hi, is possible to add option to verify more seeds at once from file? I´m try to recover old wallet, but one word in sentence is bad. for now. I have a full 2048 seed list, but I´m unable to find check script, for find correct seed. I want to offline script, which do only one thing:
Calculate wrong, and good seed in once. Thank You for reply. J.

**Comment by LizardX2** [comment]
Hello, i recommend you to simply do it manually, its just 2048 words after all, if the wallet has a good amount of assets it would definitely be worth the time....Use such wallet as exodus which is quite fast on resetting seeds.

**Comment by CryptoRecoveryWorldwideCZ** [comment]
I have it already, bud for You is easy to do it. iam W user, and Python is for everyone bad,

**Comment by CryptoRecoveryWorldwideCZ** [comment]
Some skils we maidat

**Comment by LizardX2** [comment]
"BTCMiner" won't help you with that as it might false flag correct seed and say i thas no balance, (the tool currently checks a single address path and format so it won't check all the addresses that might be on the seed....Do it manually

**Comment by CryptoRecoveryWorldwideCZ** [comment]
nope, You dont understand. I have text file with 2048 seeds, and only one thing, which I want is, recognize valid seeds., and output it to fie, or in console. Your script have ony one check.

**Comment by LizardX2** [comment]
Oh got it, you need to see documentation of how the seeds are validated.

**Comment by CryptoRecoveryWorldwideCZ** [comment]
ok, thanks
